### ISSUE TITLE
[BUG] Blocked aria-hidden on a <input> element when opening the dropdown for the first time.

### ISSUE THREAD

**Issue by AleixFerreCP** [bithost-gmbh/ngx-mat-select-search#474]
**Describe the bug**

Only the first time you open the dropdown, the console shows an error. See the [logs](https://github.com/user-attachments/files/16469252/stackblitz.com-1722594377838.log) for the full detailed error.

**To Reproduce**
1. Go to [the template](https://stackblitz.com/github/bithost-gmbh/ngx-mat-select-search-example?file=src%2Fapp%2Fapp.component.html) 
2. Open the console (from the browser, not the stackblitz console)
3. Click to open the dropdown
4. See the error appearing only the first time you open the dropdown. It does not appear later even if you change the values selected.

**Expected behavior**

This error should not appear on the console and it should be accessible by default.

**Screenshots**

https://github.com/user-attachments/assets/ccd05885-872f-4b49-830c-a632d3c39aab

**Desktop (please complete the following information):**
 - OS: macOS 14.5
 - Browser: Google Chrome
 - Version: 127.0.6533.89 (Official compilation) (arm64) 

**Smartphone (please complete the following information):**
Not tested

**Comment by macjohnny** [comment]
@AleixFerreCP thanks for reporting! do you want to fix it?

**Comment by AleixFerreCP** [comment]
> @AleixFerreCP thanks for reporting! do you want to fix it?

I can't fix it myself right now, but I'd appreciate it if you could take a look when you can.

**Comment by macjohnny** [comment]
The problem is this line: 

https://github.com/bithost-gmbh/ngx-mat-select-search/blob/443195f54288c61bdf58aecd4c810b647d32c120/src/app/mat-select-search/mat-select-search.component.ts#L288

however, this is needed to avoid the mat-option from being announced as an option. not sure whats the best way to solve this

**Comment by AleixFerreCP** [comment]
You can try to set the role of it to [presentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/presentation_role).
Although I did not test it, I think this could work.

```ts
 this.matOption._getHostElement().setAttribute('role', 'presentation');
```

**Comment by AleixFerreCP** [comment]
Yesterday, I published a simple PR to fix this issue. Can you check it out, please?

**Comment by macjohnny** [comment]
Released in [v7.0.7](https://github.com/bithost-gmbh/ngx-mat-select-search/releases/tag/7.0.7)
